js/ui/html_editor/modules/tableResizing.js

```diff
@@ -0,0 +1,524 @@
+import { getWindow, hasWindow } from '../../../core/utils/window';
+import { extend } from '../../../core/utils/extend';
+
+const DX_COLUMN_RESIZE_FRAME_CLASS = 'dx-table-resize-frame';
+const DX_COLUMN_RESIZER_CLASS = 'dx-htmleditor-column-resizer';
+const DX_ROW_RESIZER_CLASS = 'dx-htmleditor-row-resizer';
+const DEFAULT_MIN_COLUMN_WIDTH = 40;
+const DEFAULT_MIN_ROW_HEIGHT = DEFAULT_MIN_COLUMN_WIDTH / 2;
+
+const DRAGGABLE_ELEMENT_OFFSET = 2;
+
+const MODULE_NAMESPACE = 'dxHtmlTableResizingModule';
+
+const POINTERDOWN_EVENT = addNamespace('dxpointerdown', MODULE_NAMESPACE);
+const SCROLL_EVENT = addNamespace('scroll', MODULE_NAMESPACE);
+
+export default class TableResizingModule extends BaseModule {
+    constructor(quill, options) {
+        super(quill, options);
+        this.enabled = !!options.enabled;
+        this._tableResizeFrames = [];
+        this._minColumnWidth = options.minColumnWidth ?? DEFAULT_MIN_COLUMN_WIDTH;
+        this._minRowHeight = options.minRowHeight ?? DEFAULT_MIN_ROW_HEIGHT;
+        this._quillContainer = this.editorInstance._getQuillContainer();
+        this._tableData = [];
+
+        if(this.enabled) {
+            this.editorInstance.addContentInitializedCallback(() => {
+                const $tables = this._findTables();
+                if($tables.length) {
+                    this._fixTablesWidths($tables);
+                    this._createResizeFrames($tables);
+                    this._updateFramesPositions();
+                    this._updateFramesSeparators();
+                }
+
+                this._attachEvents();
+            });
+
+            this.addCleanCallback(this.clean.bind(this));
+            this._resizeHandler = _windowResizeCallbacks.add(this._resizeHandler.bind(this));
+        }
+    }
+
+    _attachEvents() {
+        eventsEngine.on(this.editorInstance._getContent(), SCROLL_EVENT, this._updateFramesPositions.bind(this));
+
+        this.quill.on('text-change', this._getQuillTextChangeHandler());
+    }
+
+    _detachEvents() {
+        eventsEngine.off(this.editorInstance._getContent(), MODULE_NAMESPACE);
+        this.quill.off('text-change', this._quillTextChangeHandler);
+    }
+
+    _getQuillTextChangeHandler(delta, oldContent, source) {
+        return (delta, oldContent, source) => {
+            if(this._isTableChanging()) {
+
+                const $tables = this._findTables();
+                this._removeResizeFrames();
+                if(source === 'api') {
+                    this._fixTablesWidths($tables);
+                }
+
+                this._updateTablesColumnsWidth($tables);
+                this._createResizeFrames($tables);
+                this._updateFramesPositions();
+                this._updateFramesSeparators();
+            } else {
+                this._updateFramesPositions();
+            }
+        };
+    }
+
+    _getFrameForTable($table) {
+        return this._framesForTables($table.get(0));
```

Comment:
mm, it seems broken. Shouldn't it be `return this._framesForTables**.get**($table.get(0));`